src/cql/r4/Factors_to_Consider_in_Managing_Chronic_Pain_FHIRv400.cql

```diff
@@ -977,6 +985,27 @@ define ReportMostRecentMME:
       Result: QuantityText(M.value as FHIR.Quantity),
       Date:   ObservationDate(M)
     }
+}
+
+//MME Calculation
+define ActiveOpioidMedications:
+  C3F.ActiveMedicationRequest([MedicationRequest: "Opioid Pain Medications"])
+
+define CurrentMME:
+  MMECalculator.TotalMME(ActiveOpioidMedications)
```

Comment:
I *think* it is calculating total MME value for MedicationRequest(s) that has a status of *active* in this valueset
https://github.com/uwcirg/AHRQ-CDS-Connect-PAIN-MANAGEMENT-SUMMARY/blob/64b6b74f29e54dede54cf6d9736c50531af2f469/src/cql/r4/Factors_to_Consider_in_Managing_Chronic_Pain_FHIRv400.cql#L76
See definition for ActiveOpioidMedications here:  https://github.com/uwcirg/AHRQ-CDS-Connect-PAIN-MANAGEMENT-SUMMARY/blob/64b6b74f29e54dede54cf6d9736c50531af2f469/src/cql/r4/CDS_Connect_Commons_for_FHIRv400.cql#L396
Might be good to figure out if **ActiveOpioidMedications** here is returning null